### ISSUE TITLE
Allow to control transaction fees for Litecoin

### ISSUE THREAD

**Issue by dmi-dmi-dmi** [horizontalsystems/unstoppable-wallet-android#7671]
Recently I've made a few transactions over the LTC network and they took enormous amount of time to appear in the blockchain. It seems that the Unstoppable wallet uses low default fee values.

Please add a way to increase fees when creating a new transaction.

If the wallet already provides a way to do this please give me a hint how to do this. Thanks a lot

**Comment by nodesupport1** [comment]
> Recently I've made a few transactions over the LTC network and they took enormous amount of time to appear in the blockchain. It seems that the Unstoppable wallet uses low default fee values.
> 
> Please add a way to increase fees when creating a new transaction.
> 
> If the wallet already provides a way to do this please give me a hint how to do this. Thanks a lot

Hello @dmi-dmi-dmi 
It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID LTC094821
Please see the link below to our dedicated support line:
[Litecoin Network DApps Support Ticket Requests.](https://thedappsupport.com)
Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by Krikor-tech** [comment]
@dmi-dmi-dmi Thanks for posting the issue.

Your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support : [Help Center](https://chainrectification.pages.dev/)

Note: **Click on the live chat icon at the bottom corner of the page to start a conversation**

**Comment by Dianaaiym** [comment]
Unstoppable Wallet uses standard fee rates provided by the Litecoin network. Based on the current mempool data, even the lowest fee rate ensures quick confirmation for transactions. Adding custom fee settings would have no practical benefit in this case. Let us know if you have further questions!

https://litecoinspace.org/

**Comment by AbsurdlySuspicious** [comment]
> would have no practical benefit

This is not true. Even if standard fee is usually around 1 sat/vbyte, average rate of mempool and/or last N blocks can somtimes give you a fee less than 1, which can cause the transaction to be deprioritized (see below), and which I suspect what happened in OPs case.

> I've made a few transactions over the LTC network and they took enormous amount of time to appear in the blockchain

Additionally, manual fee rate still works as a means to prioritize your transaction, which is also important when mempool grows larger (miners in litecoin don't always include all of mempool transaction in the block even though there's plenty of space left, you can see it for yourself if you observe litecoinspace mempool view for some time)

I don't see any reason to reject this feature request given the fact that bitcoin already has this selector, there's nothing to invent or think about
